linkerd/core/src/test/scala/io/buoyant/linkerd/LoadBalancerTest.scala

```diff
@@ -1,6 +1,6 @@
 package io.buoyant.linkerd
 
-import com.twitter.finagle.loadbalancer.{DefaultBalancerFactory, FlagBalancerFactory, LoadBalancerFactory}
+import com.twitter.finagle.loadbalancer.{FlagBalancerFactory, LoadBalancerFactory}
```

Comment:
is `FlagBalancerFactory` actually used? does it just need to be in scope or something?

Reply:
it's used on line 27 and line 48